src/plugins/discover/public/application/components/sidebar/string_progress_bar.tsx

```diff
@@ -17,35 +17,12 @@
  * under the License.
  */
 import React from 'react';
-import { EuiFlexGroup, EuiFlexItem, EuiProgress, EuiText, EuiToolTip } from '@elastic/eui';
+import { EuiProgress } from '@elastic/eui';
 
 interface Props {
   percent: number;
-  count: number;
 }
 
-export function StringFieldProgressBar(props: Props) {
-  return (
-    <EuiToolTip
-      anchorClassName="dscProgressBarTooltip__anchor"
-      content={props.count}
-      delay="regular"
-      position="right"
-    >
-      <EuiFlexGroup alignItems="center" responsive={false}>
-        <EuiFlexItem>
-          <EuiProgress
-            value={props.percent}
-            max={100}
-            color="secondary"
-            aria-hidden={true}
-            size="l"
-          />
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiText size="xs">{props.percent}%</EuiText>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    </EuiToolTip>
-  );
+export function StringFieldProgressBar({ percent }: Props) {
+  return <EuiProgress value={percent} max={100} color="secondary" aria-hidden={true} size="s" />;
```

Comment:
I think we could replace the Tooltip that is causing troubles, and set the tooltip's text as `aria-label` here , removing `aria-hidden` here